src/RageThreads.h

```diff
@@ -2,6 +2,9 @@
 #define RAGE_THREADS_H
 
 #include <cstdint>
+#include <limits>
+
+static const std::uint_fast64_t FAST64MAX = std::numeric_limits<std::uint_fast64_t>::max();
```

Comment:
Let's do FAST_ULL_MAX. Other uses of uint64 specify it as ULL so we should do the same.